packages/nimble-components/.storybook/blocks/story-layout.css

```diff
@@ -15,6 +15,7 @@
 }
 
 .story-layout-frame {
+    margin-bottom: 32px !important;
```

Comment:
Adds a bit of spacing to the bottom of our component layout frames within MDX docs.

<img width="260" alt="Screenshot 2024-01-31 at 8 56 25 AM" src="https://github.com/ni/nimble/assets/1458528/ac8d9b92-6fac-4cdd-ad8a-0ac91327a6af">
<img width="264" alt="Screenshot 2024-01-31 at 8 56 13 AM" src="https://github.com/ni/nimble/assets/1458528/4e36e20f-8261-44b7-a2fa-f868d49e4c80">